spartan/terraform/deploy-release/main.tf

```diff
@@ -34,7 +34,7 @@ data "terraform_remote_state" "metrics" {
   backend = "gcs"
   config = {
     bucket = "aztec-terraform"
-    prefix = "metrics-deploy/us-west1-a/aztec-gke-private/metrics/terraform.tfstate"
+    prefix = "metrics-deploy/us-west1-a/aztec-gke-private/${var.METRICS_NAMESPACE}/terraform.tfstate"
```

Comment:
derived from this in metrics deployment:

          terraform init \
            -backend-config="bucket=${{ env.TF_STATE_BUCKET }}" \
            -backend-config="prefix=metrics-deploy/${{ env.REGION }}/${{ env.CLUSTER_NAME }}/${{ env.NAMESPACE }}/terraform.tfstate"